src/components/CIPipelineN/ConditionContainer.tsx

```diff
@@ -337,9 +348,8 @@ export const ConditionContainer = ({ type }: { type: ConditionContainerType }) =
                                                       }
                                                     : null
                                             }
-                                            tabIndex={1}
                                             placeholder="Select variable"
-                                            onChange={(selectedValue) => {
+                                            onChange={(selectedValue: OptionType) => {
```

Comment:
extract this function out

Reply:
It is extracted out already, To pass index have to do this way.